### PR TITLE
Factor out open license check

### PR DESCRIPTION
**Pull request by jodygarnett**
In response to https://github.com/metadata101/iso19139.ca.HNAP/issues/97

### CODE REVIEW

File: src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-open-license.sch [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <xsl:sequence
+            select="
+            every $listValue in $list
+            satisfies exists($searchStrings[. = $listValue])
+            "
+    />
+  </xsl:function>
+
+  <!-- ===================================================================
+  EC schematron rules for open-data license validation in metadata editor:
```
Comment (ianwallen) [inline src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-open-license.sch:68]: Don't believe this is and EC rule.

**Review by ianwallen — COMMENTED** [review]
How is this fixing your issue where another agency is going to have its own license?  Are you planning on creating another license schematron for them?

I know from our end, management does not like having all these schematrons as they feel it confuses the users.   They would rather have validations required for the purpose. 
i.e. we will be publishing data to FGP, Open Data, internal Internal... And the users may only know English. The reason for language specific is because they will send the other language for translation only after the initial language passes validation.
So they would like the users to validate FGP English, FGP French, Open Data English.... 
But this can end up with a lot of schematrons.... 

Lets bring this topic up at the next meetings

**Comment by jodygarnett** [comment]
Is there an include? That way we could make one check for each service .. which includes only the parts needed.

**Comment by jodygarnett** [comment]
Closing this, overriding the `rdf` file contents is working out for now.

I would still like to pursue the idea of using xml import to make these checks match different publication targets.

**Pull request closed without merging**